### DISCUSSION TITLE
Use of gammap for splitting a StaticGen into multiple dynamic ones

### DISCUSSION THREAD

**Discussion by amparocorvalan** [CURENT/andes#559] [General]
Hi everyone,

I am trying to create a model where a StaticGen distributes its load in different DynamicGen using gammap but it does not work. Could someone who has implemented it give me an example of how to do it? I don't know how to implement the subidx of the StaticGen to index the DynamicGen, when i run the power flow it doesn't read the sub index.

I'm working with ieee14 and i tried adding a new generator.

Here is how I tried to implement it:

![image](https://github.com/user-attachments/assets/0071a1b5-3c6c-4b62-8a17-2da821397f1f)

![image](https://github.com/user-attachments/assets/b48cd664-a9b5-4bb9-a5ec-be456a1a82fc)

And the error I got:
![image](https://github.com/user-attachments/assets/a66c8fef-8515-4d01-bd6c-673435176132)

**Comment by cuihantao** [comment]
If you want to distribute StaticGen with idx=5 to two dynamic generators, you won't need to use `subidx`.

Set both GENROU's `gen` to `5`, and set the `gammap`s properly to up to 1. Likewise for `gammaq`.

**Reply by amparocorvalan** [reply]
Thank you Hantao! It worked that way :)
